api/v1alpha1/validation/securitypolicy_validate.go

```diff
@@ -1,154 +0,0 @@
-// Copyright Envoy Gateway Authors
```

Comment:
This file is deleted from the API package and the validations are moved into the gatewapi translator.